src/github.com/couchbase/sync_gateway/rest/config.go

```diff
@@ -278,6 +285,9 @@ func (self *ServerConfig) MergeWith(other *ServerConfig) error {
 	if self.Facebook == nil {
 		self.Facebook = other.Facebook
 	}
+  if self.CORS == nil {
+  	self.CORS = other.CORS
+  }
```

Comment:
I'm pretty sure I ran that but never hurts to run again.

On Thursday, March 12, 2015, Adam Fraser <notifications@github.com> wrote:

> In src/github.com/couchbase/sync_gateway/rest/config.go
> <https://github.com/couchbase/sync_gateway/pull/695#discussion_r26320139>:
>
> > @@ -278,6 +285,9 @@ func (self *ServerConfig) MergeWith(other *ServerConfig) error {
> >  	if self.Facebook == nil {
> >  		self.Facebook = other.Facebook
> >  	}
> > +  if self.CORS == nil {
> > +  	self.CORS = other.CORS
> > +  }
>
> Can you run go fmt for this PR? Looks like there are a few places that
> need cleaned up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/couchbase/sync_gateway/pull/695/files#r26320139>.
>


-- 
Chris Anderson
http://www.couchbase.com
http://twitter.com/jchris
"Compromising before you even begin fighting is illogical." – Joshua Wong